app/src/main/java/de/danoeh/antennapod/dialog/ItemFilterDialog.java

```diff
@@ -76,6 +89,10 @@ public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
         return dialog;
     }
 
+    protected void setResetButtonVisibility(int visibility) {
+        resetTextView.setVisibility(visibility);
+    }
```

Comment:
Encapsulating a single method call into a function seems a bit unnecessary. I would just make the member variable protected